lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -104,7 +104,7 @@ def plugin_paths
 
       def fixup_sti_loader!
         sti_loader_yml_path = miq_dir.join("tmp/cache/sti_loader.yml")
-        sti_loader          = YAML.load_file(sti_loader_yml_path)
+        sti_loader          = YAML.unsafe_load(File.read(sti_loader_yml_path))
```

Comment:
note, `unsafe_load_file` didn't exist in all versions of psych so I use the more verbose but backward compatible code.